airbyte-integrations/bases/base-python/base_python/client.py

```diff
@@ -186,3 +186,10 @@ def streams(self) -> Generator[AirbyteStream, None, None]:
     @abstractmethod
     def health_check(self) -> Tuple[bool, str]:
         """Check if service is up and running"""
+
+
+def configured_catalog_from_client(client: BaseClient) -> ConfiguredAirbyteCatalog:
```

Comment:
where is this used?

Reply:
I use this one manually to generate configured_catalog for integrational tests